src/host.js

```diff
@@ -32,14 +32,24 @@ export default class Host extends Port {
 			this.use(hostSyncLang);
 		}
 		this.use(hostSyncTitle({page: options.syncPageTitle ? true : false}));
-		if(options.resizeFrame !== false) {
-			this.use(resizer);
+
```

Comment:
If we set the height below when the iframe is created, this could become:

    if(height && options.resizeFrame !== false) {
      this.use(resizer);
    }